pkg/types/mounter.go

```diff
@@ -28,7 +28,7 @@ type Mounter interface {
 }
 
 func NewMounter(binary string) Mounter {
-	return mount.New(binary)
+	return mount.NewWithoutSystemd(binary)
```

Comment:
This is a new method provided with the updated interface which provides a mount wrapper that does not pretend to detect systemd. For instance this has impact on `make test-cli` target being executed in a desktop env. In that case every time a new mounter got initiated a systemd dialog appeared to provide the root password interactively, even it we are not mounting anything in our unit tests and they all can run as a non root user.


```
func mount.NewWithoutSystemd(mounterPath string) mount.Interface

NewWithoutSystemd returns a Linux specific mount.Interface for the current system. 
It provides options to override the default mounter behavior. mounterPath allows
using an alternative to `/bin/mount` for mounting. Any detection for systemd functionality
is disabled with this Mounter.

```

A part from that we were already a bit outdated in that library.